Add send-verify-email endpoint to admin-rest-api

behana opened the pull request:
Add endpoint to admin-rest-api to let clients trigger verification emails.

stianst commented:
Have you created an associated JIRA issue? I can't find one.

Also, please add a test to "org.keycloak.testsuite.admin.UserTest"

behana commented:
I have added a test for send-verify-email. There wasn't a test for reset-password-email, so I have added one too. The first commit also adds a resetPasswordEmail(@QueryParam clientId) to the admin-client's interface to let clients pass a client id different from 'account' to reset-password-email as I have done it for send-verify-email. Should I create two (or even three) seperate Jira and PRs for that?

stianst commented:
Create a separate JIRA for the addition of client-id, but you can keep it in this PR as it'll only be a small change.

Commit:
KEYCLOAK-1352 Add send-verify-email to admin-rest-api

Commit:
KEYCLOAK-1353 Option to pass client_id to resetPasswordMail
